### ISSUE TITLE
Allow choosing the download by filename

### ISSUE THREAD

**Issue by wolfgangkarall** [MestreLion/humblebundle#4]
Sometimes there are multiple entries of the same type:

```
$ ./humblebundle -p windows  --download ednaandharvey_harveysneweyes  -f /home/humbledownload/downloads
2015-01-28 16:11:31,633 ERROR           Too many download candidates for 'Edna & Harvey: Harvey's New Eyes' [ednaandharvey_harveysneweyes]. Improve criteria to narrow it down.
        'Download'      2.1 GB  harveysneweyes_win_1383805789.exe
        'Download'      2.1 GB  Harveys_New_Eyes_PC_FULL_2.0_Multi_Daedalic_noDRM.exe
```

I can get around this by running with --loglevel debug and manually extracting the URL, download that with wget/curl (while manually setting the needed Cookie), but it'd be nice to have a simpler way available. :)

In all the cases where this happened, both files were of equal size and had matching md5sums, so maybe de-duplicate based on that?

**Comment by MestreLion** [comment]
Oh my, what is Humble Bundle doing? 2 downloads with exactly the same name in the same platform and architecture! Well, I guess server filename is indeed the only feasible criteria to narrow it down.

I'll add a `--server-file|-F` option to `download`, how about it?

Would this be an acceptable fix for #3 too?

**Comment by wolfgangkarall** [comment]
Yes, maybe that is the better idea anyway.
